phoenix-core/src/main/java/org/apache/phoenix/jdbc/PhoenixDriver.java

```diff
@@ -148,21 +160,23 @@ public PhoenixDriver() { // for Squirrel
     }
 
     private Cache<ConnectionInfo, ConnectionQueryServices> initializeConnectionCache() {
-        Configuration config = HBaseFactoryProvider.getConfigurationFactory().getConfiguration();
+        Configuration config = HBaseFactoryProvider.getConfigurationFactory()
+            .getConfiguration();
         int maxCacheDuration = config.getInt(QueryServices.CLIENT_CONNECTION_CACHE_MAX_DURATION_MILLISECONDS,
             QueryServicesOptions.DEFAULT_CLIENT_CONNECTION_CACHE_MAX_DURATION);
         RemovalListener<ConnectionInfo, ConnectionQueryServices> cacheRemovalListener =
             new RemovalListener<ConnectionInfo, ConnectionQueryServices>() {
                 @Override
                 public void onRemoval(RemovalNotification<ConnectionInfo, ConnectionQueryServices> notification) {
                     String connInfoIdentifier = notification.getKey().toString();
-                    LOGGER.debug("Expiring " + connInfoIdentifier + " because of "
-                        + notification.getCause().name());
+                    if (LOGGER.isDebugEnabled()) {
```

Comment:
Do we need if check here?

Reply:
thanks review, pheonix checkstyle do have, but logger use by this in industry.